sisl/viz/plotly/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+====================================
+Visualization utilities (:mod:`sisl.viz`)
+====================================
+
+.. module:: sisl.viz.plotly
+   :noindex:
+
+Classes that are used for visualizing simulations results
+
+Range routines
+==============
+
+Miscellaneous routines
+======================
+
+"""
+try:
+
+    from ._presets import *
+    from ._templates import *
+    from ._user_customs import import_user_plots, import_user_presets, import_user_sessions, import_user_plugins
+
+    from .plot import Plot, Animation, MultiplePlot, SubPlots
+    from .plots import *
+    from .session import Session
+    from .sessions import *
+    from .plotutils import load
+    from ._plotables import register_plotable
+
+    from ._express import sx as express
+
+except ModuleNotFoundError as e:
+    raise e
```

Comment:
This raise will propagate upwards and make sisl unloadable if one does not have `plotly`, I think it should *silently* continue. We don't want to print stuff for users who don't care about the visualisation module.

Reply:
What about users that do want to use the visualization module? It won't work and they won't understand why.

Reply:
Yeah, true.

If we do lazily loaded viz, then it is ok to raise the exception.